kml/serdes_binary.hpp

```diff
@@ -266,6 +277,21 @@ class DeserializerKml
         m_header.m_stringsOffset = m_header.m_compilationsOffset;
       }
     }
+    if (m_header.m_version == Version::V9)
+    {
+      // Check if file has Opensource V9 or MapsMe V9 format.
+      // Actual V9 format has 6 offsets (uint64_t) in header. While V9MM has 5 offsets.
+      // It means that first section (usually categories) has offset 0x28 = 40 = 5 * 8.
+      if (m_header.m_categoryOffset == 0x28 || m_header.m_bookmarksOffset == 0x28 ||
+          m_header.m_tracksOffset == 0x28 || m_header.m_stringsOffset == 0x28 ||
+          m_header.m_compilationsOffset == 0x28)
+      {
+        LOG(LWARNING, ("KMB file has version V9MM"));
```

Comment:
Why is it a warning? Maybe info or no log at all?

Reply:
Reduced log level to `Info`